scooby/report.py

```diff
@@ -192,7 +195,7 @@ def __init__(
             optional = ['numpy', 'scipy', 'IPython', 'matplotlib', 'scooby']
 
         PythonInfo.__init__(self, additional=additional, core=core, optional=optional, sort=sort)
-        self.ncol = int(ncol)
+        self.ncol = 2
```

Comment:
I don't think ncol should be hardcoded here, doesn't render this the ncol input parameter useless?